v3/src/components/graph/models/graph-model-utils.ts

```diff
@@ -29,7 +29,8 @@ function setPrimaryRoleAndPlotType(graphModel: IGraphContentModel) {
   // the primaryRole, or, lastly, the newly assigned place
   primaryRole = attributeType === 'numeric' ? oldPrimaryRole
     : otherAttributeType === 'numeric' ? otherAttrRole
-      : attributeType !== 'empty' ? oldPrimaryRole : otherAttrRole
+      : otherAttributeType === 'date' ? otherAttrRole
+        : attributeType !== 'empty' ? oldPrimaryRole : otherAttrRole
```

Comment:
The logic here is subtle, but I think perhaps what is desired is something more like:
```typescript
  primaryRole = ['numeric', 'date'].includes(attributeType) ? oldPrimaryRole
    : ['numeric', 'date'].includes(otherAttributeType) ? otherAttrRole
      : attributeType !== 'empty' ? oldPrimaryRole : otherAttrRole
```